src/components/Simulering/__tests__/Simulering.test.tsx

```diff
@@ -1157,4 +1157,77 @@ describe('Simulering', () => {
       expect(screen.getByText('beregning.highcharts.ingress')).toBeVisible()
     })
   })
+
+  describe('Gitt at simuleringen er i avansert visning', () => {
+    it('viser tittel og riktig ingress', async () => {
+      render(
+        <Simulering
+          visning="enkel"
+          isLoading={false}
+          headingLevel="3"
+          alderspensjonListe={alderspensjonData.alderspensjon}
+          afpPrivatListe={afpPrivatData.afpPrivat}
+          showButtonsAndTable={true}
+          aarligInntektFoerUttakBeloep="500 000"
+        />,
+        {
+          preloadedState: {
+            api: {
+              /* @ts-ignore */
+              queries: {
+                ...fulfilledGetPerson,
+              },
+            },
+            userInput: {
+              ...userInputInitialState,
+              samtykke: true,
+              currentSimulation: { ...currentSimulation },
+            },
+          },
+        }
+      )
+      expect(screen.getByText('beregning.highcharts.title')).toBeVisible()
+      expect(screen.getByText('beregning.highcharts.ingress')).toBeVisible()
+    })
+
+    it('viser MaanedsbeloepAvansertBeregning komponenten når alle betingelsene er oppfylt', async () => {
+      render(
+        <Simulering
+          visning="avansert"
+          isLoading={false}
+          headingLevel="3"
+          alderspensjonListe={alderspensjonData.alderspensjon}
+          afpPrivatListe={afpPrivatData.afpPrivat}
+          showButtonsAndTable={true}
+          aarligInntektFoerUttakBeloep="500 000"
+        />,
+        {
+          preloadedState: {
+            api: {
+              /* @ts-ignore */
+              queries: {
+                ...fulfilledGetPerson,
+              },
+            },
+            userInput: {
+              ...userInputInitialState,
+              samtykke: true,
+              currentSimulation: { ...currentSimulation },
+            },
+          },
+        }
+      )
+
+      expect(await screen.findByTestId('highcharts-done-drawing')).toBeVisible()
+
+      // Nødvendig for at animasjonen rekker å bli ferdig
+      await act(async () => {
+        await new Promise((r) => setTimeout(r, 500))
+      })
+
+      expect(
+        screen.getByTestId('maanedsbloep-avansert-beregning')
+      ).toBeVisible()
```

Comment:
Ser ut som du kan klare deg med dette:
```suggestion
      expect(
        await screen.findByTestId('maanedsbloep-avansert-beregning')
      ).toBeVisible()
```